src/constants/locConstants.ts

```diff
@@ -548,6 +548,36 @@ export let reloadChoice = l10n.t("Reload Visual Studio Code");
 export let switchToMsal = l10n.t("Switch to MSAL");
 export let dismiss = l10n.t("Dismiss");
 export let querySuccess = l10n.t("Query succeeded");
+export let searchObjectsPlaceholder = l10n.t("Search for database objects...");
```

Comment:
@croblesm, can you please go over these display strings and see if they look fine.